tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/AutoInstrumentation/AWS/Shared/SpanPointersTests.cs

```diff
@@ -116,6 +116,7 @@ public void AddDynamoDbSpanPointer_ShouldAddCorrectSpanLink()
         var stringAttr = new Mock<IDynamoDbAttributeValue>();
         stringAttr.Setup(a => a.S).Returns("testvalue");
         var keys = new Mock<IDynamoDbKeysObject>();
+        keys.Setup(k => k.Instance).Returns(new object()); // not important of what it returns as long as it isn't null
```

Comment:
Done due to the new (correct) check if the keys.Instance is null

tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Shared/SpanPointers.cs